_data/nav.yml

```diff
@@ -1068,50 +1068,50 @@
 
 - url: "changelog"
   pages:
-  - url: "#Version 5.7.1 March 17, 2021"
-  - url: "#Version 5.7.0 February 10, 2021"
-  - url: "#Version 5.6.2 December 8, 2020"
-  - url: "#Version 5.6.1 November 25, 2020"
-  - url: "#Version 5.6.0 November 18, 2020"
-  - url: "#Version 5.5.1 October 1, 2020"
-  - url: "#Version 5.5.0 September 29, 2020"
-  - url: "#Version 5.4.2 August 17, 2020"
-  - url: "#Version 5.4.1 July 8, 2020"
-  - url: "#Version 5.4.0 June 30, 2020"
-  - url: "#Version 5.3.2 June 10, 2020"
-  - url: "#Version 5.3.1 May 27, 2020"
-  - url: "#Version 5.3.0 May 21, 2020"
-  - url: "#Version 5.2.2 April 23, 2020"
-  - url: "#Version 5.2.1 March 25, 2020"
-  - url: "#Version 5.2.0 February 13, 2020"
-  - url: "#Version 5.1.6 January 28, 2020"
-  - url: "#Version 5.1.5 December 19, 2019"
-  - url: "#Version 5.1.4 December 11, 2019"
-  - url: "#Version 5.1.3 December 4, 2019"
-  - url: "#Version 5.1.2 November 19, 2019"
-  - url: "#Version 5.1.1 October 28, 2019"
-  - url: "#Version 5.1.0 October 17, 2019"
-  - url: "#Version 5.0.16 September 24, 2019"
-  - url: "#Version 5.0.15 September 2, 2019"
-  - url: "#Version 5.0.14 August 19, 2019"
-  - url: "#Version 5.0.13 August 6, 2019"
-  - url: "#Version 5.0.12 July 18, 2019"
-  - url: "#Version 5.0.11 July 4, 2019"
-  - url: "#Version 5.0.10 July 2, 2019"
-  - url: "#Version 5.0.9 June 26, 2019"
-  - url: "#Version 5.0.8 June 18, 2019"
-  - url: "#Version 5.0.7 June 5, 2019"
-  - url: "#Version 5.0.6 May 22, 2019"
-  - url: "#Version 5.0.5 May 9, 2019"
-  - url: "#Version 5.0.4 April 23, 2019"
-  - url: "#Version 5.0.3 March 19, 2019"
-  - url: "#Version 5.0.2 March 5, 2019"
-  - url: "#Version 5.0.1 February 21, 2019"
-  - url: "#Version 5.0.0 February 4, 2019"
-  - url: "#Version 5.0.0-rc-2 January 22, 2019"
-  - url: "#Version 5.0.0-rc-1 January 8, 2019"
-  - url: "#Version 5.0.0-beta-1 November 30, 2018"
-  - url: "#Version 5.0.0-preview-4 November 12, 2018"
-  - url: "#Version 5.0.0-preview-3 October 18, 2018"
-  - url: "#Version 5.0.0-preview-2 October 10, 2018"
-  - url: "#Version 5.0.0-preview-1 October 1, 2018"
+  - url: "#5.7.1 - 2021-03-17"
```

Comment:
We definitely won't lose any SEO points by chaining this around will we?

Reply:
no idea. the old format wasn't idea for SEO anyway... as stated above, we'll hard-code better titles at a later date (I don't want to learn ruby just for this task).